recipes/configs/generate.yaml

```diff
@@ -0,0 +1,32 @@
+
+# Model arguments
+model:
+  _component_: torchtune.models.llama2.llama2_13b
+
+checkpointer:
+  _component_: torchtune.utils.FullModelHFCheckpointer
+  checkpoint_dir: /tmp/Llama-2-13b-hf/
+  checkpoint_files: [
+    pytorch_model-00001-of-00003.bin,
+    pytorch_model-00002-of-00003.bin,
+    pytorch_model-00003-of-00003.bin
+  ]
+  recipe_checkpoint: null
```

Comment:
Is this pretty much always going to be null? We don't really need to pass in recipe state into generate, IIUC?

Reply:
Oh yeh, good point. I should remove this